backend/services/implementations/__tests__/testService.test.ts

```diff
@@ -43,6 +45,56 @@ describe("mongo testService", (): void => {
     }).rejects.toThrowError(`userId ${mockTest.admin} not found`);
   });
 
+  it("updateTest", async () => {
+    // insert test into database
+    const createdTest = await MgTest.create(mockTest);
+
+    // mock response of user service
+    userService.getUserById = jest.fn().mockReturnValue(mockAdmin);
+
+    // create DTO object to update to
+    const testUpdate = {
```

Comment:
can we move `testUpdate` to `testUtils/tests.ts` so we can reuse? (can name it smth like `updatedTest`)

Reply:
I feel like this logic is specific to the test case - if we embed it in a different file it might be hard to understand the test case. Let me know your thoughts

Reply:
hm, i don't think it will lose too much clarity - maybe adding a comment with it can help? i just think its nice to try to keep our test data organized in one file. if you feel strongly otherwise though, we can leave as is too!